frontend/src/components/Navbar.jsx

```diff
@@ -67,7 +66,7 @@ const Navbar = () => {
                 )}
                 <div className="relative group">
                   <button className="flex items-center hover:text-blue-200 transition-colors">
-                    {user?.user.nom || 'Mon compte'}
+                    {user?.nom || 'Mon compte'}
```

Comment:
```suggestion
                    {user?.name || 'Mon compte'}
```
Why not convention ? @elhalj

Reply:
it's possible but on the model of backend i used "nom". but i'll change in "name" on the backend